build.sbt

```diff
@@ -45,18 +47,18 @@ lazy val magnum = project
     Test / fork := true,
     publish / skip := false,
     libraryDependencies ++= Seq(
-      "org.scalameta" %% "munit" % "0.7.29" % Test,
+      "org.scalameta" %% "munit" % "1.0.2" % Test,
       "com.dimafeng" %% "testcontainers-scala-munit" % testcontainersVersion % Test,
       "com.dimafeng" %% "testcontainers-scala-postgresql" % testcontainersVersion % Test,
-      "org.postgresql" % "postgresql" % "42.6.0" % Test,
+      "org.postgresql" % "postgresql" % "42.7.4" % Test,
```

Comment:
Can you also update the docker images to the latest LTS release? For example in PgTests:464

```scala
  val pgContainer = ForAllContainerFixture(
    PostgreSQLContainer
      .Def(dockerImageName = DockerImageName.parse("postgres:15.2"))
      .createContainer()
  )
```

We can use the 17.0 image instead.

Reply:
Done 🙂

Reply:
Thanks, please also update the docker images for the other Dbs as well:

* ClickHouseTests:393 Version: 24.3.12.75
* MySqlTests:395 Version: 8.4.2
* OracleTests:391 Switch to use https://hub.docker.com/r/gvenzl/oracle-free and version 23.5-faststart (or just 23.5 if that one doesn't work)
* Also, PgCodecTests:157 in magnum-pg module to 17.0

Reply:
@AugustNagro Done :)

Reply:
Thanks, getting

```
java.lang.IllegalStateException: Failed to verify that image 'gvenzl/oracle-free:23.5-faststart' is a compatible substitute for 'gvenzl/oracle-xe'. This generally means that you are trying to use an image that Testcontainers has not been designed to use. If this is deliberate, and if you are confident that the image is compatible, you should declare compatibility in code using the `asCompatibleSubstituteFor` method. For example:
   DockerImageName myImage = DockerImageName.parse("gvenzl/oracle-free:23.5-faststart").asCompatibleSubstituteFor("gvenzl/oracle-xe");
```

With the new Oracle image.. would you mind reverting to the earlier gvenzl/oracle-xe? It's not up-to-date with the latest release but there's no rush if test-containers doesn't support it yet.

Reply:
MySQL tests are failing for mysql:8.4.3, with

```
Could not create/start container
```

Looking at the docker logs, it exits because of a removed option:

```
unknown option '--skip-host-cache'
```

This was [reported in testcontainers-java](https://github.com/testcontainers/testcontainers-java/issues/8130) and fixed in [1.20.2](https://github.com/testcontainers/testcontainers-java/commit/9de26f11b0904c376ae367db8c715ca2d9d12e9d#diff-7c905dac89a79f57898f057c7dbcdd937bd27c4b9839fce5d7978ac90c76bdf5R14)

However, testcontainers-scala depends on v 1.19.x .

I've filed an issue to upgrade: https://github.com/testcontainers/testcontainers-scala/issues/368

For now, lets just revert the MySQL image upgrade as well.

Reply:
@AugustNagro Can we merge https://github.com/AugustNagro/magnum/pull/53 maybe, please? That will help me detect these stuff

Reply:
I reviewed #53 and left a few comments.

Just fyi it's very easy to run the tests locally. You just need to install docker (docker desktop or docker engine), then run `sbt test`.